.travis.yml

```diff
@@ -35,7 +35,10 @@ matrix:
 
   include:
     - php: 7.0
-      env: PHPCS=1 DEFAULT=0
+      env: PHPCS=1 DEFAULT=0 PHPSTAN=1
```

Comment:
This will make phpstan check run along with the phpcs check. It would be better to have the phpstan check in a job of it's own so that it can be added to "allow_failures" for now. It will be a while before we can make all required changes to code to make it pass.